content/docs/forms.md

```diff
@@ -155,7 +155,7 @@ class FlavorForm extends React.Component {
   }
 
   handleSubmit(event) {
-    alert('Your favorite flavor is: ' + this.state.value);
+    alert('आपका पसंदीदा रंग है: ' + this.state.value);
     event.preventDefault();
   }
 
```

Comment:
option values can be translated as well

Reply:
Made some changes. 
I believe this was similar to changes required at the dropdown's values level (Grapefruit ->चकोतरा ) and not the option tag's value property.